SU-SWS/Stanford-Drupal-Profile
# Stable20170525

# READY TO MERGE

# Summary
- These commits bring Drupal-Profile module versions in line with stable20170525.  I also updated stanford_person and stanford_landing_page to match those versions currently in use by the stanford-jumpstart-deployer and required by their associated tests.

# Needed By (Date)
- No real world deadline, but the sooner the better, as too many feature branches risk re-doing work.

# Steps to Test
- In addition to these tests passing in Travis, double check that all changes you made to https://github.com/SU-SWS/Stanford-Drupal-Profile/pull/82 have also been made here.  After this gets merged, we should be able to delete PR 82.

# Affected Projects or Products
- These updates relate to https://github.com/SU-SWS/stanford-jumpstart-deployer/pull/146.

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/DEVOPS-16
